app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,302 @@
+If you have a place on a teacher training course where you pay a fee, you do not need to apply for the international relocation payment (IRP). Your training provider will email you if your choice of course makes you eligible for the IRP. Your visa status and date of arrival will be checked by the Department for Education. If you are eligible, you’ll get the IRP around the end of your first term.
+
+If you believe you meet the eligibility requirements, but have not been informed by your teacher training provider that you may be awarded the IRP, please contact your training provider directly first. If your training provider cannot help you, you can email us at teach.inengland@education.gov.uk.
+
+### Salaried trainees
+
+If you have a place on a teacher training course where you are paid a salary to work while you train, email us at IRP.ExpressInterest@education.gov.uk to let us know you are interested in applying for the IRP. You will be asked to fill in a form giving us your details and we’ll get in touch when IRP applications open for salaried trainees in autumn 2023. If you are eligible, you’ll get the IRP around the end of your first term.
+
+## Criteria for teachers  
+
+To be eligible for the international relocation payment (IRP), you must meet all 3 of the following requirements.
+
+### 1. Teaching subject (teachers)
+
+You must have accepted a job as a teacher of languages or physics in an English state secondary school on a contract lasting at least one academic year.  
+
+Teachers of all languages offered in English state secondary schools are eligible to apply for the IRP. Language(s), with the exception of English language, can be combined with another subject, but must make up at least 50% of teaching time. You can, however, combine teaching English language with another language (for example, French, German or Spanish), as long as the non-English language makes up at least 50% of your teaching timetable.  
+
+Physics can be taught with another subject, but must make up at least 50% of teaching time. Teachers of general science or combined science  are also eligible to apply for the IRP, as long as they are teaching the physics element of general science or combined science. General science or combined science can be taught with another subject, but must make up at least 50% of teaching time.  
+
+Check with your employing school if you’re not sure whether your teaching timetable meets the teaching subject requirements.
+
+'State schools' are schools funded by the UK government. 'Secondary schools' teach children aged 11 to 16 or 11 to 18. You will not be eligible if you teach in a school for pupils aged 16 to 18 only.  
+
+An academic year is 3 full school terms: for example, if your teaching job started in September 2023 (the start of the autumn term), it would need to continue until at least July 2024 (covering the spring and summer terms). ‘Ongoing’ or ‘permanent’ contracts lasting longer than 1 academic year are also eligible.
```

Comment:
```suggestion
An academic year is 3 full school terms: for example, if your teaching job started in September 2023 (the start of the autumn term), it would need to continue until at least July 2024 (covering the spring and summer terms). Ongoing or permanent contracts lasting longer than 1 academic year are also eligible.
```
do we need quote marks here?

Reply:
Probs not